commons/src/main/java/com/afollestad/materialdialogs/color/ColorChooserDialog.java

```diff
@@ -569,16 +572,16 @@ private Builder getBuilder() {
     return (Builder) getArguments().getSerializable("builder");
   }
 
-  private void dismissIfNecessary(AppCompatActivity context, String tag) {
-    Fragment frag = context.getSupportFragmentManager().findFragmentByTag(tag);
+  private void dismissIfNecessary(FragmentManager fragmentManager, String tag) {
+    Fragment frag = fragmentManager.findFragmentByTag(tag);
     if (frag != null) {
       ((DialogFragment) frag).dismiss();
-      context.getSupportFragmentManager().beginTransaction().remove(frag).commit();
+      fragmentManager.beginTransaction().remove(frag).commit();
     }
   }
 
   @NonNull
-  public ColorChooserDialog show(AppCompatActivity context) {
+  public ColorChooserDialog show(FragmentManager fragmentManager) {
```

Comment:
I don't think changing the signature of a widely used method is a good idea. It'd be better to add an overload of `show()`.